tools/install.sh

```diff
@@ -32,9 +32,9 @@ sed -i -e "/export PATH=/ c\\
 export PATH=\"$PATH\"
 " ~/.zshrc
 
-if [ "$SHELL" != "$(which zsh)" ]; then
+if [ "$SHELL" != "$(grep /zsh$ /etc/shells | tail -1)" ]; then
```

Comment:
Good point, we should take this chance to put zsh as the default runner and implement the check for installed zsh that so many people have asked already.